runtime/router/router.go

```diff
@@ -153,3 +158,14 @@ func (r *Router) allowed(path, reqMethod string) string {
 
 	return strings.Join(allow, ", ")
 }
+
+func (r *Router) isWhitelistedPath(path string) bool {
+	if len(r.WhitelistedPaths) > 0 {
```

Comment:
*nit*: I don't think this `if` check is necessary, the `for` loop adequately handles this.

Reply:
will update it.